### ISSUE TITLE
Request: Copy pasting

### ISSUE THREAD

**Issue by gccltl** [runelite/runelite#3793]
**Is your feature request related to a problem? Please describe.**

OSRS Accounts usually are valuable, to protect valuable things people tend to use password managers, the problem is that OSRS does not allow copy-pasting passwords into the fields.

**Describe the solution you'd like**

Manipulate text fields, so it's possible to copy-paste.

**Describe alternatives you've considered**

-
**Additional context**

-

**Comment by ypperlig** [comment]
You can use a password manager like KeePass and use the autotyper for the time being. There is PR that implements this functionality, but judging from the last Discord discussion this is unlikely to get implemented in the near future.

**Comment by deathbeam** [comment]
Can we stop requesting people to change password managers whenever they ask this? lol. Noone would do that if they have their preferred one.

**Comment by ypperlig** [comment]
I don't see anything wrong in providing people an alternative until the PR is eventually merged. For all we know their password manager may have an auto typer they are unaware of.

**Comment by deathbeam** [comment]
It's like telling people to change operating system to fix some rendering issues.

**Comment by gccltl** [comment]
I'm using BitWarden, which does not have any auto typing features. Also, regular KeePass is so ugly - it's unusable, and the only fork with good design (KeeWeb) has this feature removed due to security issues. It's nice to hear that there's some PR to implement this!

**Comment by douglasg14b** [comment]
This is a pretty make or break issue for most of us that use password managers. There is no way I'm manually typing in my password dozens of times a day. I was trying to ditch OSBuddy, but it looks like I need to go back :/

**Comment by douglasg14b** [comment]
Ah! Thanks, that kind of info is good to have on closed issues like these as a closing message. As people will get here from google and not see a resolution.

A side note. The login screen plugin was already on, it needed the "Ctrl-V Paste" option enabled.

**Comment by deathbeam** [comment]
It has resolution, if you click on the link that closed this issue

**Comment by douglasg14b** [comment]
Hi @deathbeam 

It's good form to have the resolution in the same thread, it's not good for future visitors to have to hunt around for it, hoping they can find it in a PR or some other nest of threads. Never mind the non-technical users who don't really have the ability to do that.

You're essentially writing documentation with these threads, as this is where search results will lead users to. It should be treated as such imho.

**Comment by deathbeam** [comment]
The issue has resolution, it is linked to pull request:

```
@Adam- Adam- closed this in #2827 on Nov 19, 2018
```

**Comment by douglasg14b** [comment]
Oh, I meant the specific actionable item for users. Not the linked pull request or other threads.

This:
```
Turn on the login screen plugin

EDIT: and make sure the ctrl-v paste option is enabled
```
